parser_library/test/workspace_manager_test.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "gtest/gtest.h"
 
+#include "config/pgm_conf.h"
```

Comment:
not needed?

Reply:
Still needed because of line 113 where `load_from_json` is called